packages/core/src/tracing/instrumentation/protocols/httpClient.js

```diff
@@ -199,7 +201,8 @@ function instrument(coreModule, forceHttps) {
     }
 
     cls.ns.run(() => {
-      const span = cls.startSpan('node.http.client', constants.EXIT, parentSpan.t, parentSpan.s);
+      // NOTE: added type-safety in case of parentSpan is undefined or null
```

Comment:
> // NOTE: added type-safety in case of parentSpan is undefined or null

// NOTE: added type-safety in case of `allowExitRootSpan` being set to true